Changes

```diff
@@ -451,6 +451,9 @@ OCaml 4.14.0
   operation in `NO_SHARING` mode.
   (Xavier Leroy, report by Anil Madhavapeddy, review by Alain Frisch)
 
+- #10907, #10959: Wrong type inferred from existential types
+  (Jacques Garrigue and Gabriel Scherer, report by @dyzsr *)
```

Comment:
Nitpick: There is an extra `*` at the end. Is this muscle-memory due to writing too many comments? (I would expect François Pottier to make these typos out of habit, but not Jacques :-D)

Reply:
Well, this PR is pretty short, and it still includes a comment (in a mli, I admit), so this is not so true :-)